### PR TITLE
remove png from metadata attributes and beautify JSON output

### PR DESCRIPTION
**Pull request by victoryeo**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -34,10 +34,12 @@ const main = async (numberOfOutputs) => {
                         }).concat({trait_type: traitType, value: 'N/A'})
                       ));
 
+  //console.log(traitTypes)
```
Comment (victorquanlam) [inline index.js:37]: please remove console.log

File: index.js [hunk 2]
```diff
@@ -34,10 +34,12 @@ const main = async (numberOfOutputs) => {
...
   const backgrounds = fs.readdirSync(dir.background);
 
   // trait type avail for each punk
   const combinations = allPossibleCases(traitTypes,numberOfOutputs)
+  //console.log(combinations)
```
Comment (victorquanlam) [inline index.js:42]: please remove console.log

File: index.js [hunk 3]
```diff
@@ -104,13 +106,24 @@ const drawImage= async (traitTypes, background, index) => {
 
   console.log(`Progress: ${index+1}/ ${totalOutputs}`)
 
+  //deep copy
```
Comment (victorquanlam) [inline index.js:109]: great!!

**Review by victorquanlam — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (victoryeo) [reply index.js]: console.log is removed.

File: index.js (hunk 2 above)
Comment (victoryeo) [reply index.js]: console.log is removed.

File: index.js (hunk 1 above)
Comment (victorquanlam) [reply index.js]: Thanks

**Pull request merged**